packages/peregrine/lib/store/actions/cart/asyncActions.js

```diff
@@ -451,6 +451,7 @@ function isInvalidCart(error) {
         error.graphQLErrors.find(
             err =>
                 err.message.includes('Could not find a cart') ||
+                err.message.includes("The cart isn't active") ||
```

Comment:
Hmm - how does this retry on product add? `useProductFullDetail` invokes the add mutations directly and just returns on error.

https://github.com/magento/pwa-studio/blob/develop/packages/peregrine/lib/talons/ProductFullDetail/useProductFullDetail.js#L247-L253